apps/console/src/extensions/configs/models/common.ts

```diff
@@ -24,6 +24,7 @@ export interface CommonConfig {
         enableQuerySearch: boolean;
     };
     blockLoopBackCalls: boolean;
+    checkCustomLayoutExistanceBeforeEnabling: boolean;
```

Comment:
Let's update the cloud extension configs as well.

Reply:
Sure, once this is released will update the cloud extensions with the bump for the new version.